controllers/forum.js

```diff
@@ -7,6 +7,14 @@ const logger = require('../config/logger');
  */
 exports.create = async (req, res) => {
   const doc = await Forum.create(req.body);
+  if (req.body.isAnswer) {
+    const forum = await Forum.findByIdAndUpdate(req.body.parentId, { $push: { answers: doc._id } });
+    logger.info(
+      `Created answer ${doc._id} to question ${forum._id} posted by user ${req.body.author}`
+    );
+    return res.status(201).json(doc);
```

Comment:
No need to return here, since it returns below on line 19.

Reply:
I did that as I'm logging different info, otherwise below it will log once again

Reply:
That's fine I believe, you can just change the log message on line 12 to indicate that you've linked the answer to a question, and also created a doc for the answer.